veggieseasons/lib/screens/search.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/widgets.dart';
+
+class SearchScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: Text('Search'),
+      ),
+      backgroundColor: Color(0xfff0f0f0),
+      child: Center(
+        child: Text('Not yet implemented.'),
```

Comment:
personal preference alert -- I'm not sure how big you expect these different files to get, but having a separate file for *every* class feels very Java-y. But you know best how these are going to grow. Just something to think about.

Reply:
You can see the full versions in https://github.com/redbrogdon/veggieseasons, if you want. They're not huge, but big enough that I figured separating them was the way to go.